app/jobs/check_job.rb

```diff
@@ -0,0 +1,25 @@
+class CheckJob < ApplicationJob
+  queue_as :default
+
+  def perform(check, batch: nil, callback_uri: nil)
```

Comment:
Rather than a single batch as an argument, it'd probably more efficient to have a query that pulls out all batches that are affected by a link